src/tree_schema_helpers.c

```diff
@@ -1315,6 +1315,204 @@ lysp_find_module(struct ly_ctx *ctx, const struct lysp_module *mod)
     return NULL;
 }
 
+enum yang_keyword
```

Comment:
Not sure, but would it be possible to merge this with the function from YANG parser? Isn't the set of keywords (almost) same?